book/website/communication/presentations/presentations-personal-stories.md

```diff
@@ -0,0 +1,252 @@
+(cm-presentations-personal-stories)=
+# Personal stories - Presenting Outside of Your Discipline
+
+We would like to thank **[Yanina Bellini Saibene](https://yabellini.netlify.app/)** for answering our questions.
+
+Find out more about her work here:
+* ORCID: [0000-0002-4522-7466](https://orcid.org/0000-0002-4522-7466)
+* Twitter: [@yabellini](https://twitter.com/yabellini)
+
+Yanina is a researcher at [INTA in La Pampa, Argentina](https://inta.gob.ar/personas/bellini.yanina), applying data science to the agricultural sector.
+She has a degree in Information Systems and a Master in Data Mining and Knowledge Management.
+She is the co-founder and organizer of multiple collaborative spaces such as Metadocencia, RLadies and LatinR among others.
+She has organized more than 70 events (of regional and international scope), including being general chair of the Jornadas Argentinas de Informática (2020-2021), founder and current chair of LatinR (one of the largest Latin American conferences on the use of R in research and development) and one of the global coordinators of useR! 2021.
+Both in her role as a researcher and within the R community, she works to make science as reproducible and open as possible using open informatics tools.
+
+## 1. When did you start giving talks about your research in non-academic events?
+
+I work at [INTA](https://www.argentina.gob.ar/inta) (National Institute of Agricultural Technology), which is a scientific institution.
+Our activities are organized into research & development and knowledge transfer & extension, both with the same weight and importance.
+Everything that has to do with teaching, outreach, communicating, and transferring knowledge of our production to social actors in the agricultural sector and the general public is part of our work.
+Within INTA, training and communication activities, whether through a course, a workshop, an article, or an interview in a newspaper, radio, or television, are important and necessary.
+We only make sense if people use what we do.
```

Comment:
This sentence isn't entirely clear to me: We only make sense or our activities only make sense?